concatFunctions/index.js

```diff
@@ -1,33 +1,50 @@
-
 var concat = require('concat');
 var chalk = require('chalk');
+var glob = require('glob');
+var async = require('async');
 
 function getFileExtension(file) {
     return file.substr((~-file.lastIndexOf(".") >>> 0) + 2);
 }
 
 function errorHandler(error) {
-    if(error) {
+    if (error) {
         throw new Error(error);
     }
 }
 
 function getDestination(output, files) {
-    if(!Array.isArray(files)) errorHandler('Files should be an Array'); 
+    if (!Array.isArray(files)) errorHandler('Files should be an Array');
     return (output === 'all') ? output + '.' + getFileExtension(files[0]) : output;
 }
 
 function concatFiles(files, output) {
     var destination = getDestination(output, files);
-    concat(files, destination, function(concatError) {
-        errorHandler(concatError);
-        console.log(chalk.green('Files concatenated!'));
+
+    globFiles(files, function (globFiles) {
+        concat(globFiles, destination, function (concatError) {
+            errorHandler(concatError);
+            console.log(chalk.green('Files concatenated!'));
+        });
+    });
+}
+
+function globFiles(files, next) {
+    var tasks = files.map(function (file) {
+        return function (callback) {
+            glob(file, {realpath: true}, callback);
+        }
+    });
+    async.parallel(tasks, function (error, files) {
```

Comment:
Is it really necessary to include such big dependency because of this single call? Using`glob.sync` should be just fine because actual `concat` wait has to wait for this anyway. I don't think that such tiny performance improvement matters that much unless of course someone wants to concat vast amount of different globs.